src/libcrun/utils.c

```diff
@@ -66,6 +66,8 @@
 
 #define MAX_READLINKS 32
 
+const char *context_type="context";
```

Comment:
could we avoid the global and pass it as argument to `add_selinux_mount_label`?

Reply:
Yup. But I need to pass it everywhere.